bin/anthology/papers.py

```diff
@@ -452,17 +442,17 @@ def as_markdown(self, concise=False):
         """Return a Markdown-formatted entry."""
         title = self.get_title(form="text")
 
-        authors = ""
-        for field in ("author", "editor"):
-            if field in self.attrib:
-                people = [person[0] for person in self.get(field)]
-                num_people = len(people)
-                if num_people == 1:
-                    authors = people[0].last
-                elif num_people == 2:
-                    authors = f"{people[0].last} & {people[1].last}"
-                elif num_people >= 3:
-                    authors = f"{people[0].last} et al."
+        authors = "N.N."
```

Comment:
Why "N.N."?

Reply:
It's either that, or adding a `if not authors: ...` check later in this function to format citations without authors differently. Otherwise you end up with strings like "Title (, Year)" (Example: https://aclanthology.org/W19-8700/) The bibkey generation already uses "nn" when the name field is empty, so I chose that here, but I have no strong opinion.